src/main/target/OMNIBUSF4/config.c

```diff
@@ -51,5 +76,17 @@ void targetConfiguration(void)
 #ifdef EXUAVF4PRO
     targetSerialPortFunctionConfig(targetSerialPortFunction, ARRAYLEN(targetSerialPortFunction));
 #endif
+
+#ifdef SYNERGYF4
+    barometerConfigMutable()->baro_hardware = BARO_NONE;
```

Comment:
Are the `barometerConfigMutable()->baro_hardware = BARO_NONE` and `compassConfigMutable()->mag_hardware =  MAG_NONE` actually needed to make this hardware work? If not, they should be omitted.

Reply:
Hi, since the SYNERGYF4 has no mag or baro hardware, we wanted to make sure that its set to none by default to reduce confusion. If you insist, I guess it could be left out, but we would prefer it to be like this if possible. Thanks.

Reply:
> but we would prefer it to be like this if possible

Why?

Reply:
I'll look this up when I get home, but I am under the assumption that these settings reflect on the configurator UI as both baro and mag switches being in the off (or disabled) position when doing a clean flash. I might be wrong though, it's been long since I made this, forgive me if I'm incorrect. I'll get back to you asap.

Reply:
Hi, I managed to check it on the go, so yeah the original idea was to set them to none I guess because the configurator will reflect the values as such: 

```
$('input[id="baroHardwareSwitch"]').prop('checked', SENSOR_CONFIG.baro_hardware !== 1);
$('input[id="magHardwareSwitch"]').prop('checked', SENSOR_CONFIG.mag_hardware !== 1);
```

And later we decided to add the condition on the baro and mag hardware defines in the target.h file, which (quickly judging from the compass and barometer config init code) would essentially default back to baro_none and mag_none due to no sensor defines. I'll do a final check tonight and push the necessary changes if you insist! Thanks for the review!

Reply:
I don't think users will be confused by these switches being on - this is what they are used to from every other target, and this is how the autodetection was designed to work.
Adding unnecessary configuration to the target specific configuration drives up the effort required to maintain target configurations, and should be avoided.

Reply:
I completely agree with you, I removed the sets in [6857da2](https://github.com/betaflight/betaflight/pull/7690/commits/6857da2ae2f973bffe576f47ae1ac17627430090) thanks again!